addons/l10n_tr_nilvera/lib/nilvera_client.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime
+from pprint import pformat
+import requests
+import logging
+
+from odoo import _
+from odoo.exceptions import UserError, ValidationError
+
+_logger = logging.getLogger(__name__)
+
+
+class NilveraClient:
+    def __init__(self, environment=None, api_key=None, timeout_limit=None):
```

Comment:
Last thing I forgot, I promise I'm done after ahah

`timout_limit` is actually never used anywhere, should it be removed ?
If you prefer to keep it, then it should slightly more restricted something like `min(timout_limit, 30)` to avoid someone setting it as 999999999999999999